app/src/main/java/com/android/sample/ui/createRecipe/RecipeListInstructionsScreen.kt

```diff
@@ -0,0 +1,270 @@
+import com.android.sample.resources.C.Dimension.CreateRecipeListInstructionsScreen.ROUNDED_CORNER_SHAPE
+import com.android.sample.resources.C.Dimension.CreateRecipeListInstructionsScreen.ROW_SIZE
+import com.android.sample.resources.C.Tag.RECIPE_NAME_BASE_PADDING
+import com.android.sample.resources.C.Tag.RECIPE_NAME_BUTTON_HEIGHT
+import com.android.sample.resources.C.Tag.RECIPE_NAME_BUTTON_WIDTH
+import com.android.sample.resources.C.Tag.SMALL_PADDING
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.EDIT_INSTRUCTION_ICON
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.INSTRUCTION_LIST
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.INSTRUCTION_LIST_ITEM
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.INSTRUCTION_TEXT
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.INSTRUCTION_TEXT_IN_CARD
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.INSTRUCTION_TEXT_SPACE
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.INSTRUCTION_TIME
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.NEXT_STEP_BUTTON
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.RECIPE_LIST_INSTRUCTIONS_SCREEN_SPACER1
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.RECIPE_LIST_INSTRUCTION_ICON
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.RECIPE_LIST_ITEM_THUMBNAIL
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.RECIPE_NAME_TEXT
+import com.android.sample.resources.C.TestTag.CreateRecipeListInstructionsScreen.SCREEN_COLUMN
+import com.android.sample.ui.navigation.NavigationActions
+import com.android.sample.ui.navigation.Route
+import com.android.sample.ui.navigation.Screen
+import com.android.sample.ui.utils.PlateSwipeScaffold
+
+@Composable
+fun RecipeListInstructionsScreen(
+    createRecipeViewModel: CreateRecipeViewModel,
+    navigationActions: NavigationActions
+) {
+
+  PlateSwipeScaffold(
+      navigationActions = navigationActions,
+      selectedItem = Route.CREATE_RECIPE,
+      showBackArrow = true,
+      content = { paddingValues ->
+        RecipeListInstructionsContent(
+            createRecipeViewModel = createRecipeViewModel,
+            modifier = Modifier.padding(paddingValues = paddingValues),
+            navigationActions = navigationActions,
+        )
+      })
+}
+
+@Composable
+fun RecipeListInstructionsContent(
+    createRecipeViewModel: CreateRecipeViewModel,
+    modifier: Modifier,
+    navigationActions: NavigationActions,
+) {
+
+  Column(
+      modifier = modifier.padding(RECIPE_NAME_BASE_PADDING).testTag(SCREEN_COLUMN),
+  ) {
+    RecipeProgressBar(currentStep = CURRENT_STEP)
```

Comment:
Adding comments to explain the purpose of each `Text()` component might enhance the code's readability. It would make it easier for others to understand the structure at a glance.

Reply:
Done :)